### PR TITLE
Update URL of mag_compensation.py

### PR DESCRIPTION
**Pull request by elifBalci**
Updated the url of mag_compensation.py.

### CODE REVIEW

File: en/advanced_config/compass_power_compensation.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Performing this power compensation is advisable only if all the following statem
    - Slowly lower the throttle down to zero
    - Disarm the vehicle
    > **Note** Perform the test carefully and closely monitor the vibrations.
-1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/master/src/lib/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
+1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/2c3441aa905c0a0cc34aac3850f809be3e12a5e0/src/modules/sensors/vehicle_magnetometer/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
```
Comment (RicardoM17) [inline en/advanced_config/compass_power_compensation.md:38]:
```suggestion
1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/master/src/modules/sensors/vehicle_magnetometer/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
```

@elifBalci the link I sent you on Slack works but in fact for the docs we should be pointing to the latest version of the script.

**Review by RicardoM17 — CHANGES_REQUESTED** [review]
Thanks for the contribution @elifBalci , just a small fix required.

File: en/advanced_config/compass_power_compensation.md (hunk 1 above)
Comment (elifBalci) [reply en/advanced_config/compass_power_compensation.md:38]: Right, thanks for the fix. @RicardoM17

**Review by RicardoM17 — APPROVED** [review]
Thank you once again @elifBalci 

@hamishwillee lgtm.

**Comment by hamishwillee** [comment]
Thanks guys. Merged!

**Pull request merged**